app/packages/core/src/components/Common/Checkbox.tsx

```diff
@@ -58,6 +58,7 @@ const Checkbox = <T extends unknown>({
     () => (typeof count === "number" ? constSelector(count) : null),
     [count]
   );
+  const isFilterMode = useRecoilValue(fos.isSidebarFilterMode);
```

Comment:
nit: what do you think about passing this down as prop so we keep this Checkbox component as generic as possible and configure it through props.

Reply:
will setup an `omitAggregation` prop as the flag

Reply:
Omitting the `countAtom` prop has the same effect